docs/A11y-Testing-Cross-Device.md

```diff
@@ -0,0 +1,12 @@
+# Testing on Multiple Devices for an [A11y](https://www.a11yproject.com/about/#what-does-the-term-a11y-mean) Swarm
+When developing any feature for this project we ensure we check our site is as accessible to as many users as we can by completing an [a11y swarm](https://bbc.github.io/accessibility-news-and-you/guides/accessibility-swarms.html). When conducting an A11y swarm, we often want to test a branch of Simorgh across multiple devices, e.g. on an A11y laptop with the [Assistive Technology we support](https://bbc.github.io/accessibility-news-and-you/accessibility-and-supported-assistive-technology), such as screen readers. 
```

Comment:
Update to: When developing any feature for this project we ensure we check our site is as accessible to as many users as we can. We often need to test a branch of Simorgh across multiple devices e.g. on mobile device and an a11y Windows laptop with the assistive technology we support, such as screen readers.

Reply:
Not sure this is updated @andrewscfc ?